frontend/src/stores/schedule-store.ts

```diff
@@ -191,13 +194,14 @@ export const useScheduleStore = defineStore('schedules', () => {
 
   /**
    * Converts a time (startTime or endTime) to the user's timezone from utc
-   * @param {string} time
+   * BaseTime is needed for daylight savings awareness. If your time is coming from the backend's schedule route
+   * then ensure you pass in schedule.time_updated as the baseTime!
    */
-  const timeToFrontendTime = (time) => {
+  const timeToFrontendTime = (time: string, baseTime: string) => {
     const dateFormat = DateFormatStrings.QalendarFullDay;
     const user = useUserStore();
 
-    return dj(`${dj().format(dateFormat)}T${time}:00`)
+    return dj(`${dj(baseTime).format(dateFormat)}T${time}:00`)
```

Comment:
How does this change anything, if `time` is added separately here? Sorry, I guess I too dumb to see it atm 😂

Reply:
Sorry I should have called it baseDate, but it inits the dj instance as baseTime's datetime object. And then formats it out to a string, but part of the string format is the different time...from the time variable.

I hope that made sense 😅

Reply:
Yes, I just thought it wouldn't make any difference since we're doing `.format('HH:mm');` in the end anyway 😅

Reply:
Daylight savings works differently in every single datetime library, and that keeps me up at night 🫠

Reply:
Yeah. Somebody recently said, daylight saving creates more problems then it solves. I found that to be absolutely true. 😅